src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -118,13 +118,18 @@ class TypeLookupManager
 
     /**
      * Constructor
-     * @param prot Pointer to the BuiltinProtocols object.
      */
-    TypeLookupManager(
-            fastrtps::rtps::BuiltinProtocols* protocols);
+    TypeLookupManager();
 
     virtual ~TypeLookupManager();
 
+    /**
+     * Initialize the RTPSParticipantImpl BuiltinProtocols and temp ReaderProxyData and WriterProxyData
```

Comment:
This is not true, the initialization of the BuiltinProtocols calls this method, so this method cannot initialize the builtin protocols. It saves a pointer to the BuiltinProtocols, but it does not initialize them.

Reply:
Improved in [9cf4645](https://github.com/eProsima/Fast-DDS/pull/4126/commits/9cf46451cbc0cc26f54f25c7979ada4904c22109)